arch/riscv/core/thread.c

```diff
@@ -151,7 +151,6 @@ int arch_float_disable(struct k_thread *thread)
 	return 0;
 }
 
-
```

Comment:
same here, pls, revert this white space diff

Reply:
Thanks. fixed.